orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -86,6 +86,17 @@ def forward(self, model_input, x=None, y=None, z=None):
         out = self.fc2(out)
         return out
 
+class NeuralNetSimplePositionalAndKeywordArguments(torch.nn.Module):
```

Comment:
Would a model which forward takes `forward(self, key1=value1, **kwargs)` add value to the test as opposed to just named arguments?

Reply:
That is not supported, at least with this PR.
`sig = signature(module.forward)` will not be able to understand the names of the parameters.

Reply:
Did you test it?

What if we parse the user input too? Could we use `forward` args for the positional and user input for the kwargs part?

Reply:
Thanks. I think it's all doable.
Per offline discussion, I'll do this in a separate PR.